.github/workflows/api-test.yaml

```diff
@@ -0,0 +1,48 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run API Service Tests
```

Comment:
Would spotless / static analysis be a separate workflow?

Reply:
they're done right now as a part of `build`, so they are included. Actually, so is `test`, so I'll remove it for now with a comment.

Reply:
Aha, thanks, the comment is helpful.